src/app/components/table_details/table_details.js

```diff
@@ -74,6 +74,20 @@ angular
                 }
             }
 
+            function retrieveOwner(group) {
+                var owner = project.project.groups[group].owner
+                var owner_identifier
+                if (owner.name && owner.email) {
+                    owner_identifier = `${owner.name} <${owner.email}>`
+                  } else if (model.owner.name) {
+                    owner_identifier = `${owner.name}`
+                  } else if (owner.email) {
+                    owner_identifier = `${owner.email}`
+                  }
+                
+                return owner_identifier
```

Comment:
```suggestion
                const {name, email} = project.project.groups[group].owner;
                const result = [];
                if (name) result.push(name);
                if (email) {
                    const toPush = result.length > 1
                        ? `<${email}>`
                        : email;
                    result.push(toPush);
                }
                return result.join(" ");
```
- `const {name, email} =` is using destructuring to define the variables `name` and `email`, either of which will be `undefined` if they do not exist on `project.project.groups[group].owner`.
- `const result = []` arrays can be defined using `const` since their place in memory isn't changing, only their elements
- `if (name) result.push(name)` one line if statements can be used on the same line as the `if`
- `const toPush = ...` using a ternary operator to conditionally define a variable depending on if the `result` array has anything in it or not
- `return result.join(" ")` will return all elements in the `result` array (separated by a `" "` if there are more than one), and and empty string if the `result` array is empty